spec/smartcar/e2e/vehicle_spec.rb

```diff
@@ -7,7 +7,7 @@
   subject { Smartcar::Vehicle }
   before(:context) do
     client = Smartcar::Oauth.new(AuthHelper.auth_client_params)
-    url = client.authorization_url
+    url = client.authorization_url({ force_prompt: true })
```

Comment:
can reorganize this and move it into the helper method so as to not repeat it in all tests...something for later.

Reply:
makes sense